ci/infra/meta.yml

```diff
@@ -0,0 +1,458 @@
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((dataworks.docker_awscli_repository))
+            version: ((dataworks.docker_awscli_version))
+            tag: ((dataworks.docker_awscli_version))
+        inputs:
+          - name: source_ami
+        params:
+          AWS_REGION: ((dataworks.aws_region))
+          AWS_ROLE_ARN: arn:aws:iam::((dataworks.aws_management_acc)):role/ci
+          AWS_DEFAULT_REGION: ((dataworks.aws_region))
+        run:
+          path: sh
+          args:
+            - -exc
+            - |
+              export AWS_DEFAULT_REGION
+              source /assume-role
+              set +x
+              SOURCE_AMI_ID="$(cat ./source_ami/id)"
+              TIMESTAMP=$(date +%s)
+              AMI_NAME="${AMI_NAME_PREFIX}${TIMESTAMP}"
+              AMI_ID=$(aws ec2 copy-image --name ${AMI_NAME} --source-image-id ${SOURCE_AMI_ID} --source-region ${AWS_REGION} | jq -r .ImageId)
+              AMI_COPY_STATE="unknown"
+              count=0
+              set +x
+              while [[ ${AMI_COPY_STATE} != 'available' ]]; do
+                echo "waiting for the AMI copy to complete: ${count}0 passed of 600 seconds"
+                if [[ ${count} -le 60 ]]; then
+                  AMI_COPY_STATE=$(aws ec2 describe-images --image-ids ${AMI_ID} | jq -r .Images[0].State)
+                  count=$((count+1))
+                else
+                  exit 1
+                fi
+                sleep 10
+              done
+              PERMISSIONS_JSON=$(aws ec2 describe-image-attribute --image-id ${SOURCE_AMI_ID} --attribute launchPermission | jq -r '.LaunchPermissions | { LaunchPermission: { Add: .}}')
+              aws ec2 modify-image-attribute --image-id ${AMI_ID} --attribute launchPermission --cli-input-json "${PERMISSIONS_JSON}"
+              echo "AMI copy complete - ${AMI_ID} / ${AMI_NAME}"
+
+    inspector-cleanup:
+      task: inspector-cleanup
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((dataworks.docker_awscli_repository))
+            version: ((dataworks.docker_awscli_version))
+            tag: ((dataworks.docker_awscli_version))
+        inputs:
+          - name: running_instance_id
+        params:
+          AWS_ROLE_ARN: arn:aws:iam::((dataworks.aws_management_acc)):role/ci
+          AWS_DEFAULT_REGION: ((dataworks.aws_region))
+        run:
+          path: sh
+          args:
+            - -exc
+            - |
+              export AWS_DEFAULT_REGION
+              source /assume-role
+              set +x
+              INSTANCE_ID="$(cat ./running_instance_id/id)"
+              aws ec2 terminate-instances --instance-ids ${INSTANCE_ID}
+
+    get-al2-hardened-ami:
+      get: dw-al2-hardened-ami
+      trigger: true
+
+    get-al2-emr-hardened-ami:
+      get: dw-al2-emr-hardened-ami
+      trigger: true
+
+    notify:
```

Comment:
this isn't used, but probably should be

Reply:
Actually it's something I created when I wanted to build an EMR AMI from the AWS provided EMR AMI, and it seems to have leaked into here.